docs/operations_table.py

```diff
@@ -80,6 +85,8 @@ def fn_to_op_table(out_filename):
             if m is not None and hasattr(m, op_name):
                 op_string = link_formatter.format(op = op_full_name, module = module_name)
                 fn_string = link_formatter.format(op = to_fn_name(op_full_name), module = to_fn_module(module_name))
+        if op_name.lower() in removed_ops:
+            fn_string = "NA"
```

Comment:
Why `.lower()`? I think it should use either ops style OperatorName or fn style operator_name, not yet another variant.

Reply:
I'd recommend creating a function `def has_fn_variant(op_name)` or similar
```
def has_fn_variant(op_name):
    no_fn_list = ['Compose']
    return False if op_name in no_fn_list else True   
```
then use it here and in line 106